scripts/userstyles.yml

```diff
@@ -770,6 +772,17 @@ userstyles:
       app-link: "https://reddit.com"
     current-maintainers: []
     past-maintainers: [*jayylmao, *rubyowo]
+  roblox:
+    name: Roblox
+    categories: [game, entertainment]
+    icon: roblox
+    color: text
+    readme:
+      usage: |+
+        > [!NOTE]
+        > This theme supports the RoPro, RoSeal, RoGold, Roblox+, and BTRoblox third party extensions. If you experience an issue with the theme, please check if it is related to a third party plugin and mention that in the issue.
+      app-link: "https://roblox.com"
+    current-maintainers: [*genshibe]
```

Comment:
include `thatoneunoriginal` as a previous maintainer. I don't want them to go creditless for a PR you based off of.